editor/import/resource_importer_texture.cpp

```diff
@@ -523,11 +547,19 @@ Error ResourceImporterTexture::import(const String &p_source_file, const String
 				int new_width = size_limit;
 				int new_height = target_image->get_height() * new_width / target_image->get_width();
 
+				if (using_fallback_size_limit) {
+					// Only warn if downsizing occurred when the user did not explicitly request it.
+					WARN_PRINT(vformat("%s: Texture was downsized on import as its width (%d pixels) exceeded the maximum importable size of %d pixels.", p_source_file, target_image->get_width(), size_limit));
```

Comment:
Trying to make this slightly more succinct.
```suggestion
					WARN_PRINT(vformat("%s: Texture was downsized on import as its width (%d pixels) exceeded the importable size limit (%d pixels).", p_source_file, target_image->get_width(), size_limit));
```